TriblerGUI/widgets/trustgraphpage.py

```diff
@@ -231,5 +342,12 @@ def update_gui_labels(self, data):
         else:
             self.window().trust_graph_progress_bar.setHidden(False)
             self.window().trust_graph_progress_bar.setValue(bootstrap_progress)
-        status_message = "Transactions: %s | Peers: %s" % (data['num_tx'], len(data['positions']))
+
+        status_message = u"<strong style='font-size:14px'>Transactions : %s &nbsp;&nbsp; " \
```

Comment:
Why `&nbsp;` here and `SPACE` in the other locations?

Reply:
Inconsistency! Resolved